src/rascal/representations/calculator_spherical_invariants.hh

```diff
@@ -1237,7 +1109,7 @@ namespace rascal {
           InvariantsDerivative & soap_vector_gradients,
           ExpansionCoeff & expansions_coefficients,
           std::shared_ptr<StructureManager> manager) {
-    constexpr static int n_spatial_dimensions = StructureManager::dim();
+    constexpr static int NDims = StructureManager::dim();
```

Comment:
I'm honestly not a fan of this renaming back to NDims, because that could refer to any kind of dimension -- the descriptor dimension, for instance -- and the previous name made it explicit that this was _spatial_ dimensions (something that's not likely to change). Just a thought.

Reply:
renamed NDims to ThreeDims

Reply:
That wouldn't have been my first choice, but I'll take it.